code/api/src/migrations/20200824200305-AddStyleToUser.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn(
+      'users',
+      'style',
+      Sequelize.STRING
```

Comment:
Instructions for adding this new column of data type 'string' to the users database

Reply:
What's the string we're expecting `style` to return here? For example, will `style` start as `null` and change to `classic` after a user takes the style survey?

Reply:
What do the up and down properties mean/do?

Reply:
The up `code` are taken when the action (migration, seed, etc..) is run. However if you ever wanted to roll back you the down `code` is run. 
syntax for up:      `... db:seed`
syntax for down: `... db:seed:undo`